tools/setup/postgres-init-dev-db

```diff
@@ -73,8 +78,6 @@ chmod go-rw ~/.pgpass
 "$(dirname "$0")/../../scripts/setup/terminate-psql-sessions" "$USERNAME" "$DBNAME" "$DBNAME_BASE"
```

Comment:
I think you need to move the `terminate-psql-sessions` command up, since it's supposed to run just before the `DROP DATABASE`.

Reply:
Hmm, but that would also put it before `CREATE USER $USERNAME`, so I guess we also need to make that run as `root`/`postgres`?